lib/inputstreamhelper.py

```diff
@@ -630,7 +670,7 @@ def _update_widevine(self):
             current_version = wv_config['version']
         else:
             component = 'Chrome OS'
-            current_version = [x for x in wv_config if config.CHROMEOS_ARM_HWID in x['hwidmatch']][0]['version']
+            current_version = self._select_best_chromeos_image(wv_config)['version']
```

Comment:
Not sure this makes sense if `wv_config` is a single object.

Reply:
wv_config (in the ARM context) is a JSON listing.